dev/create-release/release-build.sh

```diff
@@ -258,6 +258,7 @@ if [[ "$1" == "package" ]]; then
   LFTP mkdir -p $dest_dir
   LFTP mput -O $dest_dir 'spark-*'
   LFTP mput -O $dest_dir 'pyspark-*'
+  LFTP mput -O $dest_dir 'SparkR-*'
```

Comment:
I checked http://people.apache.org/~pwendell/spark-nightly/spark-master-bin/latest/ - pyspark-* is missing in all latest dir